apps/platform/src/components/AssociationsToolkit/Table/TableBody.jsx

```diff
@@ -44,6 +44,9 @@ function TableBody({ core, expanded, cols }) {
   const isExpandedInTable = expanded[3] === prefix && flatCols.includes(expanded[1]);
 
   const handleClickAway = () => {
+    if(e.srcElement.className === "CodeMirror-hint CodeMirror-hint-active"){
```

Comment:
Hi @RobinM-code! Good catch of the bug and thanks for the fix. Could you please here fix the definition of `e` by adding it as a param of the handleClickAway function

Reply:
Good catch, thanks!